sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/EncryptResult.java

```diff
@@ -0,0 +1,55 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Represents the details of encrypt operation result.
+ */
+public class EncryptResult {
+
+    public EncryptResult(byte[] cipherText, byte[] authenticationTag, EncryptionAlgorithm algorithm) {
+        this.cipherText = cipherText;
+        this.authenticationTag = authenticationTag;
+        this.algorithm = algorithm;
+    }
+
+    /**
+     * THe encrypted content.
+     */
+    private byte[] cipherText;
```

Comment:
Do we want to make these fields final?

Reply:
This too might need to be consistent across languages, will ask the KV team for P3 and see what they say.